docs/user_guide/flower_integration/flower_job_structure.rst

```diff
@@ -74,7 +74,7 @@ Here is an example of ``pyproject.toml``, taken from :github_nvflare_link:`this
 Project Name
 ------------
 The project name should match the name of the project folder, though not a requirement. In this example, it is ``flwr_pt``. 
-Serverapp Specification
+Server App Specification
 
```

Comment:
to be consistent with ClientApp, should we add "------------------------" as well?